scripts/vet.sh

```diff
@@ -70,6 +70,9 @@ not git grep "\(import \|^\s*\)\"google.golang.org/grpc/interop/grpc_testing" --
 # - Ensure that no trailing spaces are found.
 not git grep '[[:blank:]]$'
 
+# - Ensure that no tabs are found in markdown files.
+not git grep $'\t' -- '*.md'
```

Comment:
Can/Should we do this for all files?

Reply:
do you mean prohibit tab code files? I think the IDE might be doing that already.

Reply:
I could be using an editor that doesn't do that. But looks like `gofmt` uses tabs for indentation. See: https://pkg.go.dev/cmd/gofmt. And we run `gofmt` in our vet script. So, we should be good.

Reply:
Go pretty famously uses tabs instead of spaces.  Limiting to *.md SGTM.